resource_tracker/forms.py

```diff
@@ -116,6 +116,7 @@ class ResourcePoolForm(ModelForm):
 
     tags = TagField(label="Tags",
                     required=False,
+                    help_text="Comma-separated list of tags",
```

Comment:
Add a section [after this one](https://hewlettpackard.github.io/squest/0.1/manual/resource_tracking/#link-service-catalog-instances-to-resources).
Explain what are tags.

And add a copy of this [tab]( https://django-taggit.readthedocs.io/en/latest/forms.html#tags-in-forms)